### PR TITLE
Update roslyn to stable

### PR DESCRIPTION
**Pull request by ericstj**
Currently mirrored Internal build 20160613.1 to dotnet-core

### CODE REVIEW

**Comment by cston** [comment]
@tmat

File: build-info/dotnet/roslyn/netcore1.0/Latest_Packages.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-Microsoft.Net.Compilers.netcore 1.3.0-beta1-20160610-04
-Microsoft.DiaSymReader.PortablePdb 1.1.0-beta1-60610-04
+Microsoft.CodeAnalysis.Common 1.3.0
+Microsoft.CodeAnalysis.Workspaces.Common 1.3.0
+Microsoft.CodeAnalysis.CSharp 1.3.0
+Microsoft.CodeAnalysis.CSharp.Workspaces 1.3.0
+Microsoft.CodeAnalysis.VisualBasic 1.3.0
+Microsoft.CodeAnalysis.VisualBasic.Workspaces 1.3.0
+Microsoft.CodeAnalysis.Compilers 1.3.0
+Microsoft.Net.Compilers.netcore 1.3.0
```
Comment (eerhardt) [inline build-info/dotnet/roslyn/netcore1.0/Latest_Packages.txt]: @tmat told me this isn't going stable.

**Comment by cston** [comment]
LGTM

File: build-info/dotnet/roslyn/netcore1.0/Latest_Packages.txt (hunk 1 above)
Comment (cston) [inline build-info/dotnet/roslyn/netcore1.0/Latest_Packages.txt]: This can go stable after all. It doesn't have any pre-release dependencies.

Comment (tmat) [inline build-info/dotnet/roslyn/netcore1.0/Latest_Packages.txt]: Yeah, I was mistaken. We have destkop version of the package which depends on pre-release diasymreader. Our package naming is confusing :(

**Comment by eerhardt** [comment]
Alright, merging.

**Pull request merged**